autoload/db_ui/dbout.vim

```diff
@@ -123,22 +126,46 @@ function! db_ui#dbout#yank_header() abort
   call setreg(v:register, csv_columns)
 endfunction
 
-function! s:get_cell_range(line, col) abort
+function! s:get_cell_range(cell_line_number, curpos, scheme_name) abort
+  let line = getline(a:cell_line_number)
```

Comment:
Every callsite was using `getline()`, so I changed it to specifying a number and putting `getline()` here instead.

That should help streamline any further changes that must be made to this particular routine.